source/documentation/upgrade-guide/chap-Updates_between_Minor_Releases.html.md

```diff
@@ -58,6 +60,54 @@ Updates to the oVirt Engine are released via the oVirt Project.
 
 **Important:** The update process may take some time; allow time for the update process to complete and do not stop the process once initiated.
 
+## Updating the oVirt Self-Hosted Engine and Underlying Virtualization Host(s)
+
+The process for upgrading a [Self-Hosted Engine](/self-hosted/Self-Hosted_Engine_Guide/) is slightly different as the engine is running as a VM.
+
+The following assumes that you have already deployed the Hosted Engine on your hosts and the Hosted Engine VM is running the same oVirt version as the host(s).
+
+1.  Virtualization Host: Enable global maintenance mode from one of the hosts
+```
+host ~ # hosted-engine --set-maintenance --mode=global
+```
+
+2.  Virtualization Host: Update packages (you may need to changes repository to latest version or install the latest versions metapackage e.g. ovirt-release42-4.2.1)
+```
+host ~ # yum update
+```
+
+3.  Virtualization Host: Restart vdsmd
+```
+host ~ # systemctl restart vdsmd
+```
+
+4.  Virtualization Host: Restart ha-agent and broker services
+```
+host ~ # systemctl restart ovirt-ha-broker && systemctl restart ovirt-ha-agent
+```
+
+5.  Hosted-Engine-VM: Update packages
+```
+hosted-engine ~ # yum update
```

Comment:
If you do keep this separate, not that one should update only the setup packages, e.g.:

# yum update ovirt\*setup\*

(You might want to fix also the above procedure to use asterisks. I thought we did this, might be wrong).